tce/src/applibs/LLVMBackend/LLVMBackend.cc

```diff
@@ -884,15 +884,9 @@ LLVMBackend::createPlugin(const TTAMachine::Machine& target) {
     }
 
     // NOTE: this could be get from Makefile.am
-    std::string pluginSources =
-        srcsPath + "TCERegisterInfo.cc " +
-        srcsPath + "TCEInstrInfo.cc " +
-        srcsPath + "TCEISelLowering.cc " +
-        srcsPath + "TCEDAGToDAGISel.cc " +
-        srcsPath + "TCETargetObjectFile.cc " +
-        srcsPath + "TCEFrameInfo.cc " +
-        srcsPath + "TCETargetMachinePlugin.cc " +
-        srcsPath + "TCESubtarget.cc ";
+    std::string pluginSources = srcsPath + "PluginCompileWrapper.cc " +
```

Comment:
Is this what the clang-format script produces? I think it doesn't match the original TCE style.

Reply:
This is what the format-branch.sh changed it to. But I'll change it to previous styling.